salt_stack/pillar/top.sls

```diff
@@ -1,6 +1,19 @@
+# Check for existence of scalyr.sls at the root of a pillar folder.
+{% set roots = salt['config.get']('pillar_roots:base') %}
+{% set scalyr = {'exists': false} %}
+{% for path in roots if salt['file.file_exists'](path + '/scalyr.sls') %}
+  {% if scalyr.update({'exists': true}) %}
+    # Hack to update a variable inside a loop (not supported by Jinja version
+    # used by our version of Salt).  We can update the value of a dict, though.
+  {% endif %}
+  {% break %}
+{% endfor %}
+
 base:

```

Comment:
It's interesting there is one line with ^M in this file.  I guess it works with CR LF method.

Reply:
Yeah and it didn't show up in `vim` which was strange. Normally, I see them.